staging/src/k8s.io/kms/internal/plugins/_mock/go.mod

```diff
@@ -1,27 +0,0 @@
-module k8s.io/kms/plugins/mock
-
```

Comment:
Why this is removed?

`pull-kubernetes-e2e-kind-kms` failed for this package?

https://prow.k8s.io/pr-history/?org=kubernetes&repo=kubernetes&pr=121338

Reply:
oof... sorry I missed that

mind opening a PR to add them back?